src/pybind/mgr/rbd_support/task.py

```diff
@@ -373,7 +374,6 @@ def execute_task(self, sequence):
                 else:
                     task.in_progress = True
                     self.in_progress_task = task
-                    self.update_progress(task, 0)
```

Comment:
@dillaman Don't you think we can also remove `update_progress` when handling exception?

Reply:
I was figuring that if we hit an exception after the task started I would want to know the reason (so I added the `retry_message`). The `update_progress` on the exception path won't create a new progress event unless one was already posted via the librbd progress callback.